tests/ui/extern-flag/empty-extern-arg.stderr

```diff
@@ -2,10 +2,9 @@ error: extern location for std does not exist:
 
 error: `#[panic_handler]` function required, but not found
 
-error: language item required, but not found: `eh_personality`
+error: panic unwind is not supported without std
    |
-   = note: this can occur when a binary crate with `#![no_std]` is compiled for a target where `eh_personality` is defined in the standard library
-   = help: you may be able to compile for a target that doesn't need `eh_personality`, specify a target with `--target` or in `.cargo/config`
+   = help: specify panic="abort" instead
```

Comment:
Is this always correct? `#[panic_handler]` does work on no-std, so I thought you don't have to specify `panic="abort"`. Is the previous help about the target not helpful?

Reply:
unwinding panics require the eh_personality lang item. on targets that don't support unwinding at all, panic=abort is the default so you don't have to care about eh_personality at all. but on targets that do support unwinding (that people still want to write no_std binaries for sometimes) unwinding is still the default even with no_std.